src/audio/base_fw.c

```diff
@@ -314,6 +360,11 @@ static int basefw_set_large_config(struct comp_dev *dev,
 	case IPC4_FW_CONFIG:
 		tr_warn(&basefw_comp_tr, "returning success for Set FW_CONFIG without handling it");
 		return 0;
+	case IPC4_SYSTEM_TIME:
+		tr_warn(&basefw_comp_tr, "returning success for Set SYSTEM_TIME without handling it");
```

Comment:
@ujfalusi Kernel will fail if we don't handle this. I don't really see a reason for us to set the system time, we'd have to reset the Zephyr kernel time here and I don't see a good reason to do that.

Reply:
I would lover it to tr_dbg() level. The reason for this is to have the timestamps in sync (or close to in sync) with the kernel time.
When you look at both logs with times, you might be able to see better and match the events easier if the times are in sync.
But ignoring it is also fine, it is not an error, it is how it is implemented in SOF.

Reply:
Will do this.

Reply:
presumably we could store the delta and then add it in logs?

Reply:
@lyakh It's just that the timestamps are added in Zephyr logging core, so we'd need an interface to feed this timing delta from app to the Zephyr logging subsystem. Might be doable, but I'd prefer to handle this is a follow-up task if needed.